### ISSUE TITLE
Not tracking any searches

### ISSUE THREAD

**Issue by ashemedai** [chrisparnin/docsight#3]
As you asked me, here's the issue on what I am encountering.

I'm running Chrome 26 on Windows 7 and installed DocSight from the Chrome Web Store.

After that I did some test searches that led me from Google's search pages to stackoverflow and the Android developer pages. When I click the DocSight icon in the tray top-right the entire page is empty aside from the application's chrome.

Am I doing something wrong?

**Comment by chrisparnin** [comment]
Do you association your google account with chrome?  Maybe an issue accessing the "sync" storage of Chrome.  Can you examine the "Javascript Console" and see if there are any errors present there.

Thanks.

**Comment by ashemedai** [comment]
I associate my Google account with Chrome, indeed.

With the JavaScript Console open, when I then click on the DocSight icon I get this: `Uncaught TypeError: Cannot call method 'replace' of undefined (app.js.:120)`

Or did you mean from another location?

**Comment by chrisparnin** [comment]
Great!  Looks like an instance where google search is missing the q= param.

```
  v.title = decodeURIComponent(params.q.replace(/\+/g," "));
```

Will include this fix in new version I'm pushing out later today.

**Comment by ashemedai** [comment]
I did a step through the debugger with the breakpoint set at 119, found a populated v.url that was in the form of `https://www.google.com/search?hl=en&tbs=simg:...` which indeed has no q parameter whatsoever. This is apparently a Google Images search for similar images. I use this quite a bit, by the way, for finding documentation on architecture images, hardware designs/die photos, CPUs and the likes.

**Comment by chrisparnin** [comment]
I fixed the bug in version 1.0.4,  

```
https://chrome.google.com/webstore/detail/docsight/ceacnbgdhcnofnomlkmackaennjfmnpc/details
```

Past google searches is an interesting use case, but I'm not sure how to handle that at the moment.

**Comment by ashemedai** [comment]
Just an update for you, Chris. It works for me now. Thanks!
